setupTests.js

```diff
@@ -21,6 +21,7 @@
 require('core-js/es7/reflect');
 require('intersection-observer');
 require('fake-indexeddb/auto');
+require('jest-canvas-mock');
```

Comment:
Let's maybe import it on the test file that needs it, so it's clear what exactly needs this polyfil

Reply:
moved